providers/urls.py

```diff
@@ -2,6 +2,6 @@
 import providers.api as api
 
 urlpatterns = [
-    path("providers/", api.ProvidersList.as_view()),
-    path("providers/<int:pk>", api.ProvidersDetail.as_view()),
+    path('', api.ProvidersList.as_view()),
+    path('<int:pk>', api.ProvidersDetail.as_view()),
```

Comment:
We need to add the ending slash
```suggestion
    path('<int:pk>/', api.ProvidersDetail.as_view()),
```